site/e2e/global.setup.ts

```diff
@@ -1,10 +1,19 @@
 import { expect, test } from "@playwright/test";
+import { hasFirstUser } from "api/api";
 import { Language } from "pages/CreateUserPage/CreateUserForm";
+import { setupApiCalls } from "./api";
 import * as constants from "./constants";
 import { storageState } from "./playwright.config";
 
 test("setup deployment", async ({ page }) => {
   await page.goto("/", { waitUntil: "domcontentloaded" });
+  await setupApiCalls(page);
+  const exists = await hasFirstUser();
+  if (exists) {
+    // First user already exists, abort early. All tests execute this as a dependency,
+    // if you run multiple tests in the UI, this will fail unless we check this.
+    return;
+  }
```

Comment:
```suggestion
  // First user already exists, abort early. All tests execute this as a dependency,
  // if you run multiple tests in the UI, this will fail unless we check this.
  if (exists) {
    return;
  }
```

nit: I think these sorts of comments fit better above the condition rather than inside